### ISSUE TITLE
[Bug]: Date column in excel is converted to user time zone date

### ISSUE THREAD

**Issue by mainakaich** [spreadsheetimporter/ui5-cc-spreadsheetimporter#484]
### OData Version

OData V4

### Draft

Yes

### Scenario

Fiori Elements

### Environment

BTP ABAP

### UI5 Spreadsheet Component

cc.spreadsheetimporter.v0_31_4

### What happened?

I am uploading an excel file which has date in format MM/DD/YY. My computer time zone is UTC-5 hours i.e. USA EST time zone. Now when I upload the excel file, the date field value is converted to EST time. E.g., I am attaching an excel file here. When I upload this file, in preview it is changed to 14th Feb and 29th Feb. 
[Upload Article Supersession Template 01-31-2024.xlsx](https://github.com/spreadsheetimporter/ui5-cc-spreadsheetimporter/files/14139622/Upload.Article.Supersession.Template.01-31-2024.xlsx)
![excel upload show preview](https://github.com/spreadsheetimporter/ui5-cc-spreadsheetimporter/assets/54839961/21a511c2-6389-48c7-af79-4af3615c263d)

### Relevant log output

```shell
I will share if required
```

### Spreadsheet Component Init

```js
"actions": {
                                        "spreadSheetController": {
                                            "press": "zssarticle.ext.spreadSheetController.spreadSheetController.uploadExcel",
                                            "visible": true,
                                            "enabled": true,
                                            "requiresSelection": false,
                                            "text": "Upload"
                                        }
```

### Manifest

```json
"componentUsages": {
            "spreadsheetImporter": {
                "name": "cc.spreadsheetimporter.v0_31_4"
            }
        },
        "resourceRoots": {
            "cc.spreadsheetimporter.v0_31_4": "./thirdparty/customControl/spreadsheetImporter/v0_31_4"
        },
```

**Comment by marianfoo** [comment]
Hi @mainakaich,
please send me the metadata.xml if you can to marian@marianzeis.de, that makes it easier to debug.
Thanks!

**Comment by mainakaich** [comment]
Hi Marian,

I have replied back the metadata.xml to your said email address.

Mainak Aich
Mob:-+91 9831705275

**Comment by marianfoo** [comment]
Hi @mainakaich 
please check version 0.31.5 if this fixes it for you

**Comment by mainakaich** [comment]
Hi @marianfoo 

This issue looks fixed in 0.31.5. However, I have 2 apps where I want to use this component. But currently I am not able to do so as I am getting deployment error. I have logged issue #490 for the same.

**Comment by marianfoo** [comment]
Yeah, I know. That's why I asked 😃
Just because #490 is something to-do with deployment and not the component itself.  
I'm closing for now, we can reopen if the deployed version still has errors.
